### ISSUE TITLE
Remove aria-readonly attribute from role="listbox"/role="combobox"

### ISSUE THREAD

**Issue by ggkrustev** [telerik/kendo-ui-core#1740]
### Description
According W3C [aria-readonly](https://www.w3.org/TR/wai-aria/states_and_properties#aria-readonly) cannot be combined with components with role "listbox" / "combobox". This will produce an invalid WAI-ARIA decorated component.

The solution is to remove that attribute from our code, although the listed widgets are indeed "readonly" in the context of the HTML successfull components.

Reported in TicketID: 1039730
